README.md

```diff
@@ -94,6 +94,27 @@ If you want to redeploy to the same production URL you can run `yarn vercel --pr
 
 **Hint**: We recommend connecting the project GitHub repo to Vercel so you the gets automatically deployed when pushing to `main`
 
+---
+## ⚠️ Disabling type/error checks 
+> **Warning**
+> We don't encourage to use the below options
```

Comment:
Instead of this, we might want to be like:

"Typescript is great blablabla..... but is can be daunting for newcomers..... so here we show you how you can disable.....blabla... until you get familiar with it.